src/panels/config/helpers/forms/ha-schedule-form.ts

```diff
@@ -340,6 +340,7 @@ class HaScheduleForm extends LitElement {
     });
 
     if (!isSameDay(start, end)) {
+      this[`_${day}`] = this[`_${day}`].concat();
```

Comment:
Why not just call `this.requestUpdate`?

Reply:
I believe it's not sufficient just to update, but we need to run the code inside this if statement as well:

https://github.com/home-assistant/frontend/pull/17900/files#diff-ddb21d3c355e9f20f63eab35ac6d24a1838f519152997bac96d5f52bfd4576a5R185-R197
```
    if (
      changedProps.has("_sunday") ||
      changedProps.has("_monday") ||
      changedProps.has("_tuesday") ||
      changedProps.has("_wednesday") ||
      changedProps.has("_thursday") ||
      changedProps.has("_friday") ||
      changedProps.has("_saturday") ||
      changedProps.has("calendar")
    ) {
      this.calendar.removeAllEventSources();
      this.calendar.addEventSource(this._events);
    }
 ```

Reply:
It's also possible that `requestUpdate` solves this somehow too, but I'm not very familiar with how it works.

Reply:
`this.requestUpdate(`_${day}`)` should work then

Reply:
Yep looks like it does 👍 

Thanks didn't know it could do that.